rpcserver.go

```diff
@@ -6691,6 +6691,13 @@ func marshallTopologyChange(topChange *routing.TopologyChange) *lnrpc.GraphTopol
 
 	channelUpdates := make([]*lnrpc.ChannelEdgeUpdate, len(topChange.ChannelEdgeUpdates))
 	for i, channelUpdate := range topChange.ChannelEdgeUpdates {
+
+		customRecords := marshalExtraOpaqueData(
+			channelUpdate.ExtraOpaqueData,
+		)
+		inboundFee :=
+			extractInboundFeeSafe(channelUpdate.ExtraOpaqueData)
```

Comment:
nit: fold before arguments, not before function call (the same way as the statement above).